SQL/TSQL.sublime-syntax

```diff
@@ -695,6 +695,47 @@ contexts:
       pop: 1
     - include: like-else-fail
 
+###[ USER MANAGEMENT EXPRESSIONS ]#############################################
+
+  grant:
+    - meta_prepend: true
+    - match: \b(?i:on)\b
+      scope: keyword.context.resource.tsql
+      push: qualified-identifier
+    - include: with-options
+
+  user-privileges:
+    - meta_prepend: true
+    - match: \b(?i:all(?:\s+permissions)?)\b
```

Comment:
shouldn't `permissions` be `PRIVILEGES`?
https://learn.microsoft.com/en-us/sql/t-sql/statements/grant-transact-sql?view=sql-server-ver16#syntax
https://learn.microsoft.com/en-us/sql/t-sql/statements/revoke-transact-sql?view=sql-server-ver16#syntax

and if so, I think it is already covered by https://github.com/forkeith/Packages/pull/27/files#diff-532863df530e67398448cfd2730f9f561656b1a802e5ab7605948496b603ebccR980 :)

Reply:
Just misread the BNF vs. constants, I guess. Fixed. Thanks.